test/integration/features/proxy.feature

```diff
@@ -0,0 +1,81 @@
+@proxy @minishift-only
+Feature: Minishift can run behind proxy
+  As a user I can use Minishift behind a proxy.
+  INFO: Code behind this feature will start a proxy server in background
+        on port 8181 and will try to get IP of host machine automatically
+        to set MINISHIFT_HTTP_PROXY for this feature only.
+        However one can override port number or set IP manually if needed.
+        To use custom port number run this test with environmental variable
+        INTEGRATION_PROXY_CUSTOM_PORT.       
+        To use custom IP address please set INTEGRATION_PROXY_CUSTOM_IP
+        environmental variable.
+
+  Scenario: Start behind the proxy
+   Given user starts proxy server and sets MINISHIFT_HTTP_PROXY variable
+    When executing "minishift start" succeeds
+    Then proxy log should contain "Accepting CONNECT to registry-1.docker.io:443"
+     And proxy log should contain "Accepting CONNECT to auth.docker.io:443"
+
+  Scenario: User can login to the server
+   Given Minishift has state "Running"
+    When executing "oc login --username=developer --password=developer" succeeds
+    Then stdout should contain
+     """
+     Login successful
+     """
+
+  Scenario: User can create new namespace ruby for application ruby-ex
+   Given Minishift has state "Running"
+    When executing "oc new-project ruby" succeeds
+    Then stdout should contain
+     """
+     Now using project "ruby"
+     """
+
+  Scenario: User can deploy application ruby-ex to namespace ruby
+   Given Minishift has state "Running"
+    When executing "oc new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git" succeeds
+    Then stdout should contain
+     """
+     Success
+     """
+    When executing "oc rollout status deploymentconfig ruby-ex --watch" succeeds
+    Then stdout should contain
+     """
+     "ruby-ex-1" successfully rolled out
+     """
+     And proxy log should contain "Accepting CONNECT to registry-1.docker.io:443"
+     And proxy log should contain "Accepting CONNECT to bundler.rubygems.org:443"
+     And proxy log should contain "Accepting CONNECT to rubygems.org:443"
+
+  Scenario: User can create route for ruby-ex to make it visiable outside of the cluster
+   Given Minishift has state "Running"
+    When executing "oc expose svc/ruby-ex" succeeds
+    Then stdout should contain
+     """
+     exposed
+     """
+    And status code of HTTP request to "/" of service "ruby-ex" in namespace "ruby" is equal to "200"
+    And body of HTTP request to "/" of service "ruby-ex" in namespace "ruby" contains "Welcome to your Ruby application on OpenShift"
+
+  Scenario: User can delete namespace ruby
```

Comment:
Is this really needed? How is this related to proxy? I believe this is covered somewhere else, right? The tests should be to the point and as much as possible only test what's relevant for a given feature. One might as well delete here directly.